### ISSUE TITLE
network default not found error when creating a container

### ISSUE THREAD

**Issue by sean-kang** [chef-cookbooks/docker#794]
### Cookbook version
2.13.7

### Chef-client version
12.7.2

### Platform Details
Ubuntu 14.04 in vagrant and AWS EC2

### Scenario:
Setting up a docker container fails with "network default not found" error.

### Steps to Reproduce:
Create a docker container by using the following code.

```
docker_service 'default' do
  action [:create, :start]
end

docker_image 'grafana' do
  repo 'grafana/grafana'
  tag '3.1.1'
  action :pull
end

docker_container 'grafana' do
  repo 'grafana/grafana'
  tag '3.1.1'
  port '80:3000'
  restart_policy 'always'
  action :run
end
```

### Expected Result:
Create a new container called grafana using grafana/grafana:3.1.1 image.

### Actual Result:
ERROR: docker_container[grafana] (some_wrapper::metrics_store line <line_number that points to docker_container>) had an error: Docker::Error::NotFoundError: network default not found

**Comment by sean-kang** [comment]
Possibly related to [this](https://github.com/docker/docker/pull/14530).

**Comment by sean-kang** [comment]
Specifying network mode by adding `network_mode 'bridge'` seems to work around the problem.

**Comment by chasebolt** [comment]
I have a fix incoming for this but currently running into test-kitchen issue. Once tests pass I will release.

**Comment by someara** [comment]
This should be fixed up in 2.13.8
